rsconnect/bundle.py

```diff
@@ -498,7 +498,7 @@ def write_manifest(
     return created, skipped
 
 
-def list_files(base_dir, include_sub_dirs, walk=os.walk):
+def list_files(base_dir, include_sub_dirs, walk=os.walk, followlinks=True):
```

Comment:
Just reading the diff, does this get used? Or does it belong where walk is called in L512?

Reply:
🤦

Reply:
@mmarchetti It actually looks to me like this function has never been used, except from its own unit test. Safe to delete?

If not, I think I'd need to change it to effectively `walk=functools.partial(os.walk, followlinks=True)` or equivalent.

Reply:
> Safe to delete?

Seems like it.